src/screens/explore-screen/components/ArtistCard.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useCallback } from 'react'
+
+import { User } from 'audius-client/src/common/models/User'
+import { profilePage } from 'audius-client/src/utils/route'
+import { push } from 'connected-react-router'
+import { StyleProp, ViewStyle } from 'react-native'
+
+import { Card } from 'app/components/card'
+import { useDispatchWeb } from 'app/hooks/useDispatchWeb'
+import { formatCount } from 'app/utils/format'
+
+const formatProfileCardSecondaryText = (followers: number) => {
+  const followersText = followers === 1 ? 'Follower' : 'Followers'
+  return `${formatCount(followers)} ${followersText}`
+}
+
+type ArtistCardProps = {
+  artist: User
+  style: StyleProp<ViewStyle>
+}
+
+export const ArtistCard = ({ artist, style }: ArtistCardProps) => {
+  const dispatchWeb = useDispatchWeb()
+  const goToRoute = useCallback((route: string) => dispatchWeb(push(route)), [
+    dispatchWeb
+  ])
+
+  return (
+    <Card
+      style={style}
+      id={artist.user_id}
+      imageSize={artist._profile_picture_sizes}
+      primaryText={artist.name}
+      secondaryText={formatProfileCardSecondaryText(artist.follower_count)}
+      onPress={() => goToRoute(profilePage(artist.handle))}
```

Comment:
also nit: the useCallback isnt necessary if we just inline a function with it, maybe update goToRoute, to be goProfilePage, and combine everything?